doc/source/development/rfc/rfc108_driver_removal_3_11.rst

```diff
@@ -0,0 +1,160 @@
+.. _rfc-108:
+
+=====================================================================
+RFC 108: Driver removals for GDAL 3.11
+=====================================================================
+
+============== =============================================
+Author:        Even Rouault
+Contact:       even.rouault @ spatialys.com
+Started:       2025-02-18
+Status:        Draft
+Target:        GDAL 3.11
+============== =============================================
+
+Summary
+-------
+
+This RFC communicates about the partial or total removal of obsolete drivers
+for GDAL 3.11.
+
+Motivation
+----------
+
+GDAL is an ever growing code base, with probably more than 50% of the code
+base being used by less of 1% of users. This is isn't very sustainable, and quite
+demotivating for current maintainers who know nothing/little about most drivers,
+have no evidence that they are used, and regularly stumble upon them when doing
+whole codebase enhancements.
+The decision of which driver to removal comes from a mix of "intuition" and
+community feedback from
+https://docs.google.com/spreadsheets/d/12yk0p8rRK4rAYO_VwXboatWkiILAjE2Cepu32lsHhM8/edit?gid=0#gid=0
+
```

Comment:
Why not consider available data such as [traffic to documentation pages](https://rpubs.com/dbaston/1267034)? This would show e.g. ~90 hits/month to the documentation for the TIGER driver, which is more than retained drivers such as JSONFG.

Reply:
TIGER may be a special case and hit statistics contain mishits because most TIGER data are not in that format.

JSONFG is a format of the future, not yet accepted as an OGC standard, but it will be soon. TIGER is a format from the past. The choise which one to keep is clear.

Reply:
I'm not suggesting that we remove JSONFG because it has less web traffic than TIGER. But I don't think we should disregard that traffic either. Given relatively recent [discussion](https://forum.ipums.org/t/basic-questions-about-digital-block-geometry-for-1980-to-2000-for-us-territories/3994) or [usage](https://www.mdpi.com/2571-6255/6/8/300) of data in this format, I would not write off the site traffic as accidental.

Reply:
In the "usage" link they have used `Data: County subdivision lines from US Census Bureau 2018 TIGER/line files`, thus shapefiles, but it does not mean that some other researcher wants and needs to use historical TIGER data for a good reason, like in the "discussion" link.